airflow/models/__init__.py

```diff
@@ -107,7 +107,7 @@ def __getattr__(name):
     "TaskReschedule": "airflow.models.taskreschedule",
     "Trigger": "airflow.models.trigger",
     "Variable": "airflow.models.variable",
-    "XCom": "airflow.models.xcom",
+    "XComModel": "airflow.models.xcom",
```

Comment:
This is to support  `from airflow.models importXCom` so ths should be
```suggestion
    "XCom": "airflow.sdk.execution_time.xcom",
```

Reply:
Yep added it

Reply:
Not add -- we don't need XComModel in here

Reply:
Made that change